js/gated-content/src/components/blog/BlogListing.vue

```diff
@@ -132,6 +141,20 @@ export default {
         };
       }
 
+      if (this.favorites) {
```

Comment:
We can refactor this and move to the mixin

Reply:
Based on our conversation - decided to skip this.